src/main/kotlin/org/rust/cargo/toolchain/tools/Cargo.kt

```diff
@@ -171,7 +172,14 @@ open class Cargo(toolchain: RsToolchain, useWrapper: Boolean = false)
     ): BuildScriptsInfo? {
         if (!isFeatureEnabled(RsExperiments.EVALUATE_BUILD_SCRIPTS)) return null
         val additionalArgs = listOf("--message-format", "json")
-        val commandLine = CargoCommandLine("check", projectDirectory, additionalArgs)
+        val nativeHelper = RsPathManager.procNativeHelper()
+        val envs = if (nativeHelper != null) {
```

Comment:
It would be great to have the ability to disable using native helper in case of any bug in it.
Otherwise, we can break existing feature (yep, it's experimental but anyway) without any workaround

Reply:
I've added a registry key `org.rust.cargo.evaluate.build.scripts.wrapper` (true by default)